Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -15,26 +15,12 @@
     <% if root_url != "#{HOMEBREW_BOTTLE_DEFAULT_DOMAIN}/bottles" %>
     root_url "<%= root_url %>"
     <% end %>
-    <% if ![HOMEBREW_DEFAULT_PREFIX,
-            HOMEBREW_MACOS_ARM_DEFAULT_PREFIX,
-            HOMEBREW_LINUX_DEFAULT_PREFIX].include?(prefix) %>
-    prefix "<%= prefix %>"
-    <% end %>
-    <% if cellar.is_a? Symbol %>
-    cellar :<%= cellar %>
-    <% elsif ![Homebrew::DEFAULT_MACOS_CELLAR,
-               Homebrew::DEFAULT_MACOS_ARM_CELLAR,
-               Homebrew::DEFAULT_LINUX_CELLAR].include?(cellar) %>
-    cellar "<%= cellar %>"
-    <% end %>
     <% if rebuild.positive? %>
     rebuild <%= rebuild %>
     <% end %>
-    <% checksums.each do |checksum_type, checksum_values| %>
-    <% checksum_values.each do |checksum_value| %>
-    <% checksum, macos = checksum_value.shift %>
-    <%= checksum_type %> "<%= checksum %>" => :<%= macos %>
-    <% end %>
+    <% checksums.each do |checksum| %>
+    <% cellar = checksum["cellar"] %>
+    <%= "sha256" %> "<%= checksum["digest"] %>" => :<%= checksum["tag"] %><% if cellar.is_a? Symbol %>, :cellar => :<%= cellar %><% elsif ![Homebrew::DEFAULT_MACOS_CELLAR, Homebrew::DEFAULT_MACOS_ARM_CELLAR, Homebrew::DEFAULT_LINUX_CELLAR].include?(cellar) && !cellar.blank?%>, :cellar => "<%= cellar %>"<% end %>
```

Comment:
That's a very long line. Any way we can shrink it a bit?

Reply:
Done, I added the generate_sha256_line method to do just that.